app/models/episode_import.rb

```diff
@@ -18,25 +18,37 @@ class EpisodeImport < ApplicationRecord
   scope :having_duplicate_guids, -> do
     unscope(where: :has_duplicate_guid).where(has_duplicate_guid: true)
   end
-  scope :complete, -> { where(status: PodcastImport::COMPLETE) }
+  scope :complete, -> { where(status: COMPLETE) }
+  scope :finished, -> { where(status: [COMPLETE, FAILED]) }
+  scope :in_progress, -> { where.not(status: [COMPLETE, FAILED, CREATED]) }
+  scope :failed, -> { where(status: FAILED) }
```

Comment:
different scopes needed for various UI conditionals. not sure if there was a better way to go about this but it feels cleaner here than in the views.